### DISCUSSION TITLE
I am using Keycloak v23.0.4 in kubernetes with a mysql m6g.xlarge rds instance. The K8s pod configuration involves 3 pods each with 3 core CPU and 6 GB memory. While trying to load test the Keycloak instance with 50 TPS I am getting the following error - ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (executor-thread-448) Acquisition timeout while waiting for new connection. Has anyone faced this similar issue?

### DISCUSSION THREAD

**Discussion by Alok-Kumar-9** [keycloak/keycloak#26352] [Q&A]
I am using Keycloak v23.0.4 in kubernetes with a mysql m6g.xlarge rds instance. The K8s pod configuration involves 3 pods each with 3 core CPU and 6 GB memory. While trying to load test the Keycloak instance with 50 TPS I am getting the following error - ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (executor-thread-448) Acquisition timeout while waiting for new connection. Has anyone faced this similar issue?

**Comment by shawkins** [comment]
@Alok-Kumar-9 that exception is likely due to too few database connections for the amount of load. Have you tried increasing the connection pool size?
